src/libraries/System.Net.Http/tests/FunctionalTests/LoopbackSocksServer.cs

```diff
@@ -290,16 +290,9 @@ void HandleExceptions(Exception ex)
             }
         }
 
-        private async ValueTask ReadToFillAsync(Stream stream, Memory<byte> buffer)
+        private ValueTask ReadToFillAsync(Stream stream, Memory<byte> buffer)
```

Comment:
Can we just delete this helper and have the call sites use ReadExactlyAsync directly?

Reply:
I can. It will cause a few more changes in the file, it is called 10 times. But it will clean the code up a bit.